packages/logging-bunyan/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@google-cloud/logging-bunyan",
+  "version": "0.0.0",
```

Comment:
We have a release task (`npm run publish-module [version number]`) which for the first release, we'll start at `0.1.0`